### ISSUE TITLE
is it possible to get historical data with this API?

### ISSUE THREAD

**Issue by Peppershaker** [hongtaocai/googlefinance#42]
Had a quick look at the code and im guessing the answer is no?

**Comment by kpagcha** [comment]
I don't think so. But you can build your own db downloading historical data as csv directly from Google finance and then use this lib to keep it up to date.

**Comment by jfunction** [comment]
Seems like they removed the ability to download historical data a few days ago. Anyone found work arounds?
